modules/update-service-create-service-web-console.adoc

```diff
@@ -0,0 +1,35 @@
+[id="update-service-create-service-web-console_{context}"]
+= Creating an OpenShift Update Service application by using the web console
+
+You can use the {product-title} web console to create an Update Service application by using the OpenShift Update Service Operator.
+
+.Prerequisites
+
+* The OpenShift Update Service Operator has already been installed.
+* The Openshift Update Service graph-data container image has already been created and pushed to a repository accessible to the OpenShift Update Service.
+
+.Procedure
+
+To create an OpenShift Update Service application by using the {product-title} web console:
+
+. In the {product-title} web console, click *Operators* -> *Installed Operators*.
+
+. Choose *OpenShift Update Service* from the list of installed Operators.
+
+. Click the *Update Service* tab.
+
+. Click *Create UpdateService*.
+
+. Enter a name in the *Name* field, for example, `service`.
+
+. Enter the local pullspec in the *Graph Data Image* field to the graph-data container image created in xref:modules/update-service-graph-data.adoc#update-service-graph-data_{context}[Creating the OpenShift Update Service graph data container image], for example, `registry.example.com/openshift/graph-data:latest`.
+
+. Enter the local registry and repository in the *Releases* field, for example, `registry.example.com/openshift/release`.
```

Comment:
In the **Mirroring the OpenShift Container Platform image repository** section, we create 2 repositories, LOCAL_REPOSITORY and LOCAL_RELEASE_IMAGES_REPOSITORY. We need to clearly indicate that the one contains release images is used in the **Releases** field.

Reply:
fixed